src/i18n/locales/en.json

```diff
@@ -38,10 +40,11 @@
     "Login": "Login",
     "Logout": "Logout",
     "Loyalty status": "Loyalty status",
+    "New password": "New password",
     "No results found":"No results found",
     "No time zone found": "No time zone found",
     "No warehouses found":"No warehouses found",
-    "OMS": "OMS",
```

Comment:
We need to keep this, as we are using it on the login page.

Reply:
Yes sir, done